src/tests/running.md

```diff
@@ -342,14 +342,35 @@ coordinate running tests (see [src/bootstrap/src/core/build_steps/test.rs]).
 > **TODO**
 >
 > - Is there any support for using an iOS emulator?
-> - It's also unclear to me how the wasm or asm.js tests are run.
+> - It's also unclear to me how the asm.js tests are run.
```

Comment:
Suggestion: yet this, asm.js was removed an eternity ago since the asmjs target got yeeted 😆